build_metal.sh

```diff
@@ -49,13 +56,17 @@ build_ttnn_tests="OFF"
 build_metal_tests="OFF"
 build_umd_tests="OFF"
 build_programming_examples="OFF"
+cxx_compiler_path=""
+c_compiler_path=""
 build_static_libs="OFF"
 unity_builds="ON"
+cxx_compiler_path=""
```

Comment:
Any reason for the duplicate?

Reply:
error on my part. thought this was removed already, will get a fix in soon.